### PR TITLE
dagger download: rename flag --export-path to --output

### PR DESCRIPTION
**Pull request by helderco**
With shorthand `-o` included.

### CODE REVIEW

**Review by jedevc — APPROVED** [review]
Has my vote :tada:

File: cmd/dagger/download.go [hunk 1]
```diff
@@ -30,9 +30,9 @@ var downloadCmd = &FuncCommand{
 	BeforeRequest: func(_ *FuncCommand, cmd *cobra.Command, returnType *modTypeDef) error {
 		switch returnType.ObjectName() {
 		case Directory, File, Container:
-			flag := cmd.Flags().Lookup("export-path")
+			flag := cmd.Flags().Lookup("output")
 			if returnType.ObjectName() == Container && flag != nil && !flag.Changed {
-				return fmt.Errorf("flag --export-path is required for containers")
+				return fmt.Errorf("flag --output is required for containers")
```
Comment (jedevc) [inline cmd/dagger/download.go:35]: nit: doesn't this also apply for files/directories?

**Review by sipsma — APPROVED** [review]
Worth quickly fixing Justin's comment https://github.com/dagger/dagger/pull/6026/files#r1377536301

Otherwise LGTM!

File: cmd/dagger/download.go (hunk 1 above)
Comment (helderco) [reply cmd/dagger/download.go:35]: The thing is that files and directories default to the current directory fine, so it's optional to set it. In containers you need to specify the path to a file, not a directory, that's why it's required.

**Comment by shykes** [comment]
note for follow-up change: defaulting to `.` is actually not great in practice because it pollutes the current dir. Rarely what you actually want. Would be better to default to a temporary dir.

File: cmd/dagger/download.go (hunk 1 above)
Comment (jedevc) [reply cmd/dagger/download.go:35]: I think this is linked to @shykes's comment below - if we could just default all to a temp directory, that would be a lot clearer.

**Pull request merged**